atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/KvsSweepQueueTest.java

```diff
@@ -455,4 +466,12 @@ private void assertNoEntriesInSweepableTimestampsBeforeSweepTimestamp() {
                 .nextSweepableTimestampPartition(ShardAndStrategy.conservative(CONS_SHARD), -1L, sweepTs))
                 .isEmpty();
     }
+
+    protected void putTimestampIntoTransactionTable(long ts, long commitTs) {
+        try {
+            txnService.putUnlessExists(ts, commitTs);
+        } catch (KeyAlreadyExistsException e) {
+            // this is fine
```

Comment:
No need to change this.

The duplication makes me twitch a little bit, but the comment I wrote above doesn't apply here - in this case we don't care because the commit ts is always the same.